src/SDKs/DataBox/DataBox.Tests/DataBox.Tests.csproj

```diff
@@ -5,7 +5,9 @@
     <Description>DataBox.Tests class library</Description>
     <AssemblyName>DataBox.Tests</AssemblyName>
     <VersionPrefix>1.0.0</VersionPrefix>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```

Comment:
Why is this being modified? Please undo these changes and address any warnings present

Reply:
Not sure how this automatically creaped in, taken the fix